blocks/init/src/blocks/wrapper/wrapper.php

```diff
@@ -10,36 +10,54 @@
 use Eightshift_Libs\Helpers\Components;
 
 // Used to add or remove wrapper.
-$has_wrapper = $attributes['hasWrapper'] ?? true;
+$has_wrapper        = $attributes['hasWrapper'] ?? true;
+$use_simple_wrapper = $attributes['useSimpleWrapper'] ?? false;
 
-if ( $has_wrapper ) {
+if ( ! $has_wrapper ) {
+  $this->render_wrapper_view(
+    $template_path,
+    $attributes,
+    $inner_block_content
+  );
+
+  return;
+}
 
-  $id = $attributes['id'] ?? '';
+$id = $attributes['id'] ?? '';
 
-  $wrapper_main_class = 'wrapper';
+$wrapper_main_class = 'wrapper';
 
-  $wrapper_class = Components::classnames([
-    $wrapper_main_class,
-    isset( $attributes['styleBackgroundColor'] ) && ! empty( $attributes['styleBackgroundColor'] ) ? "{$wrapper_main_class}__bg-color--{$attributes['styleBackgroundColor']}" : '',
-    $attributes['styleSpacingTop'] ? Components::responsive_selectors($attributes['styleSpacingTop'], 'spacing-top', $wrapper_main_class) : '',
-    $attributes['styleSpacingBottom'] ? Components::responsive_selectors($attributes['styleSpacingBottom'], 'spacing-bottom', $wrapper_main_class) : '',
-    $attributes['styleHideBlock'] ? Components::responsive_selectors($attributes['styleHideBlock'], 'hide-block', $wrapper_main_class, true) : '',
-  ]);
+$wrapper_class = Components::classnames([
+  $wrapper_main_class,
+  isset( $attributes['styleBackgroundColor'] ) && ! empty( $attributes['styleBackgroundColor'] ) ? "{$wrapper_main_class}__bg-color--{$attributes['styleBackgroundColor']}" : '',
```

Comment:
Why `isset()` and `! empty()` check? Usually `! empty` will suffice.

Reply:
fixed